code/modules/mob/living/carbon/alien/adult/caste/hunter.dm

```diff
@@ -75,11 +75,10 @@
 					blocked = TRUE
 			if(!blocked)
 				L.visible_message(span_danger("[src] pounces on [L]!"), span_userdanger("[src] pounces on you!"))
-				L.Paralyze(100)
-				sleep(0.2 SECONDS)//Runtime prevention (infinite bump() calls on hulks)
```

Comment:
i removed this because it's from 2010 like everything else involved with xenos

it's an indicator we haven't fucking touched this code in decades